### ISSUE TITLE
Installation on Fedora

### ISSUE THREAD

**Issue by leanderlist** [svenstaro/rofi-calc#98]
Hey, I am a noobie when it comes to building something from source. Because of that I dont really understand how I can use this awesome package on Fedora. I installed qalculate but now im lost. Could someone help me please?

**Comment by svenstaro** [comment]
The README has pretty exact steps:
```
$ git clone https://github.com/svenstaro/rofi-calc.git
$ autoreconf -i
$ mkdir build
$ cd build/
$ ../configure
$ make
$ make install
```
If you get some errors, it's likely you forgot to install some dependencies.

**Comment by leanderlist** [comment]
First of all, thanks for your reply! Sadly this didn't work. When I run the commands you told me to run I get the first error `autoreconf -i`:
aclocal: warning: couldn't open directory 'm4': No such file or directory
configure.ac:3: warning: The macro `AC_CONFIG_HEADER' is obsolete.
configure.ac:3: You should run autoupdate.
./lib/autoconf/status.m4:719: AC_CONFIG_HEADER is expanded from...
configure.ac:3: the top level
configure.ac:17: warning: The macro `AC_PROG_CC_C99' is obsolete.
configure.ac:17: You should run autoupdate.
./lib/autoconf/c.m4:1659: AC_PROG_CC_C99 is expanded from...
configure.ac:17: the top level
configure.ac:32: installing './ar-lib'
configure.ac:17: installing './compile'
configure.ac:11: installing './install-sh'
configure.ac:11: installing './missing'
Makefile.am:4: error: Libtool library used but 'LIBTOOL' is undefined
Makefile.am:4:   The usual way to define 'LIBTOOL' is to add 'LT_INIT'
Makefile.am:4:   to 'configure.ac' and run 'aclocal' and 'autoconf' again.
Makefile.am:4:   If 'LT_INIT' is in 'configure.ac', make sure
Makefile.am:4:   its definition is in aclocal's search path.
Makefile.am: installing './depcomp'
autoreconf: error: automake failed with exit status: 1

The next error occurs when i run `make`:
make: *** No targets specified and no makefile found.  Stop.

**Comment by svenstaro** [comment]
I'm not super knowledgeable about Fedora but you can try
```
sudo dnf install make automake gcc gcc-c++ kernel-devel
```

**Comment by leanderlist** [comment]
still same error

**Comment by svenstaro** [comment]
Sorry, I don't really know which packages you need on Fedora. If you do find it out, I'd be happy for a pull request!

**Comment by alienn-x** [comment]
> First of all, thanks for your reply! Sadly this didn't work. When I run the commands you told me to run I get the first error `autoreconf -i`: aclocal: warning: couldn't open directory 'm4': No such file or directory configure.ac:3: warning: The macro `AC_CONFIG_HEADER' is obsolete. configure.ac:3: You should run autoupdate. ./lib/autoconf/status.m4:719: AC_CONFIG_HEADER is expanded from... configure.ac:3: the top level configure.ac:17: warning: The macro `AC_PROG_CC_C99' is obsolete. configure.ac:17: You should run autoupdate. ./lib/autoconf/c.m4:1659: AC_PROG_CC_C99 is expanded from... configure.ac:17: the top level configure.ac:32: installing './ar-lib' configure.ac:17: installing './compile' configure.ac:11: installing './install-sh' configure.ac:11: installing './missing' Makefile.am:4: error: Libtool library used but 'LIBTOOL' is undefined Makefile.am:4: The usual way to define 'LIBTOOL' is to add 'LT_INIT' Makefile.am:4: to 'configure.ac' and run 'aclocal' and 'autoconf' again. Makefile.am:4: If 'LT_INIT' is in 'configure.ac', make sure Makefile.am:4: its definition is in aclocal's search path. Makefile.am: installing './depcomp' autoreconf: error: automake failed with exit status: 1
> 
> The next error occurs when i run `make`: make: *** No targets specified and no makefile found. Stop.

You need to install libtool

Code to append in readme : `sudo dnf install automake libtool`

**Comment by svenstaro** [comment]
Thanks for you fix in #101, @alienn-x!
